libcontainer/integration/init_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// init runs the libcontainer initialization code because of the busybox style needs
+// init runs the libcontainer initialization code because of the debian style needs
```

Comment:
Looks like this code is not really distro-dependent, so this comment should not be either.

Reply:
Thanks addressed!